ui/app/AppLayouts/Communities/panels/SharedAddressesPermissionsPanel.qml

```diff
@@ -53,7 +53,7 @@ Rectangle {
         readonly property bool lostPermissionToJoin: root.isEditMode && joinPermissionsModel.count && !joinPermissionPanel.tokenCriteriaMet
 
         readonly property var uniquePermissionChannels:
-            root.permissionsModel && root.permissionsModel.count ?
```

Comment:
Further inspecting the impact, this also improves the situation quite a bit as the `root.permissionsModel` like to change a lot while here we're only interested in the channels anyway